# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../pspresent/install.patch
# Copyright (C) 2004 - 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- pspresent-1.3/Makefile.vanilla      2007-06-19 19:51:42.634372361 +0200
+++ pspresent-1.3/Makefile      2007-06-19 19:52:14.180170054 +0200
@@ -3,4 +3,6 @@
 # Copyright (C) Matthew Chapman 2001-2003
 #

+prefix=/usr
+
 # You may need to change these paths
@@ -21,6 +21,9 @@
 $(TARGET): $(OBJS)
 	$(CC) -o $(TARGET) $(OBJS) $(LDLIBS)
 
+install:
+	install $(TARGET) $(DESTDIR)$(prefix)/
+
 clean:
 	rm $(TARGET) $(OBJS)
 
